wled00/const.h

```diff
@@ -319,6 +319,7 @@
   #define ABL_MILLIAMPS_DEFAULT 850  // auto lower brightness to stay close to milliampere limit
 #else
   #if ABL_MILLIAMPS_DEFAULT < 250  // make sure value is at least 250
```

Comment:
Please add an option to have ABL=0 which should disable ABL and not produce warning message.

Reply:
I added a check, I'm not sure that I've understood how the ABL is disabled if `ABL_MILLIAMPS_DEFAULT = 0`
by the way I added a `#if` for the check.

I'm also not sure about the 250mA lower limit value, because in "FX-fcn.cpp" at line 1063 there is a check with a value of 150:
`if (ablMilliampsMax < 150 || actualMilliampsPerLed == 0) { //0 mA per LED and too low numbers turn off calculation`

Reply:
If you do not understand something then you test all possible variations and observe the behaviour. 😄 

If the limiter is set to <150 it will not use it, effectively lighting LEDs as much as intended. It has nothing to do with ABL_MILLIAMPS_DEFAULT.